interface/themes/oemr-bs3-override.css

```diff
@@ -0,0 +1,79 @@
+/*override bootstrap 3 for openemr and some new classes all prefixed with oe-*/
+
+.form-group {
+	margin-bottom: 5px !Important;
+}
+legend {
+	border-bottom: 2px solid #E5E5E5 !Important;
+	background: #E5E5E5 !Important;
+	padding-left: 10px !Important;
+}
+.form-horizontal .control-label {
+	padding-top: 2px;
+}
+fieldset {
+	background-color: #F2F2F2!Important;
+	margin-bottom: 10px!Important;
+	padding: 0px 0px 15px 0px!Important;
+}
+.btn-link.btn-cancel:focus, .btn-link.btn-cancel:hover {
+	text-decoration: none;
+}
+.btn-link.btn-cancel {
+	border: 1px solid #CCC;
+	border-radius: 3px !Important;
+	color: #8e8e8e !Important;
+}
+.btn-link.btn-cancel.active, .btn-link.btn-cancel.focus, 
+.btn-link.btn-cancel:active, .btn-link.btn-cancel:focus,
+.btn-link.btn-cancel:hover, .open>.dropdown-toggle.btn-link.btn-cancel {
+	color: #333 !Important;
+	border-color: #ADADAD;
+	box-shadow: 0px 0px 4px #ADADAD inset;
+}
+.btn-separate-left {
+	margin-left: 20px !Important;
+}
+.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle) {
+	border-top-right-radius: 3px !Important;
+	border-bottom-right-radius: 3px !Important;
+}
+.btn-group-pinch>.btn:nth-last-child(2):not(.dropdown-toggle) {
+	border-top-right-radius: 3px !Important;
+	border-bottom-right-radius: 3px !Important;
+}
+.oe-custom-line{
+   padding-bottom:5px;
+}
+.oe-large {
+	display: inline-block;
```

Comment:
Should be more abstract, maybe called `.inline` - where is this class being used?